dev/META6.json

```diff
@@ -11,12 +11,15 @@
   "depends": [
     "Cro::HTTP",
     "Cro::HTTP::Session::Pg",
+    "Cro::HTTP::Test",
     "DB::Pg",
     "Digest::SHA1::Native",
     "OO::Monitors",
     "Shell::Command",
     "Spreadsheet::XLSX",
     "Text::CSV",
+    "Test",
```

Comment:
Hm, `Test` is provided in core, so it isn't usually mentioned in the `depends`.

Reply:
Removed